Justfile

```diff
@@ -195,6 +197,13 @@ go2proto:
 build-protos-unconditionally: pnpm-install go2proto
   cd backend/protos && buf generate
 
+# Run uv sync and generate .py and .pyi files from .proto
+build-python-protos:
+  @mk python-runtime/ftl/src/ftl/protos/schema_pb2.py python-runtime/ftl/src/ftl/protos/schema_pb2.pyi \
+      python-runtime/ftl/src/ftl/protos/console_pb2.py python-runtime/ftl/src/ftl/protos/console_pb2.pyi \
+      python-runtime/ftl/src/ftl/protos/ftl_pb2.py python-runtime/ftl/src/ftl/protos/ftl_pb2.pyi : {{PROTOS_IN}} \
+      -- "uv run protoc --proto_path=backend/protos --mypy_out=python-runtime/ftl/src/ftl/protos --python_out=python-runtime/ftl/src/ftl/protos {{PROTOS_IN}}"
```

Comment:
will buf build do this for ya instead of `uv`? https://buf.build/protocolbuffers/python?version=v28.3

Reply:
Then maybe it will just work without having a separate `build-python-protos`?